client/blb/client_test.go

```diff
@@ -257,6 +304,376 @@ func TestWriteReadTracedExactlyOneTract(t *testing.T) {
 	trace.checkLength(t, 7)
 }
 
+// setupBackupRequestFunc overrides spawning logic for backup reads, it will spawn nReaders
+// goroutines that will block on respective delayChans. To release a reader one can write to the
+// delayChan.
+func setupBackupRequestFunc(cli *Client, nReaders int) []chan time.Time {
```

Comment:
also, this is always used in conjunction with setupBackupClient, so they should probably be one function